### DISCUSSION TITLE
CPM metrics

### DISCUSSION THREAD

**Discussion by BritneyStars** [apache/skywalking#7592] [Q&A] [answered]
![segment](https://user-images.githubusercontent.com/26988781/131098246-262e0b0d-994f-4c06-a848-7fed5905bd20.jpg)
![cpm_min](https://user-images.githubusercontent.com/26988781/131098271-9a290e19-62d8-46d9-a5b7-6b3a6439f7e9.jpg)
![cpm_hour](https://user-images.githubusercontent.com/26988781/131098308-1eaf5831-c959-4552-a124-1533d981d78d.jpg)
![cpm_day](https://user-images.githubusercontent.com/26988781/131098325-7f064fdd-84f2-4a0c-9069-886314d2cc47.jpg)
As shown in FIG，the first picture shows the request total is 8174 in segment indice when statistically specified endpoint .
The secound picture shows the total request is 5999 by minute dimesion in endpoin-cpm when same contition. The third picture shows the total request is 1248 by hour dimesion in endpoin-cpm when same contition. The fourth picture shows the total request is 1389 by day dimesion in endpoin-cpm when same contition. why this? if  it does not  query the segment index ,how to get the request count in metrics-cpm indice

**Comment by wu-sheng** [comment]
I don't know what you want to say. The community would NOT do data check for your local/private env.
I am not sure why you submit this kind of question to a community.

**Reply by BritneyStars** [reply]
Ok! I only  want to kown why the total requests in segment indice and metrics-apm indice are different. I want to find the question.

**Reply by wu-sheng** [reply]
We don't know. We have run the `Accuracy Test` tests and verified. This is a distributed system, there could be many reasons, or even just your incorrect detection methods. 
But anyway, this is not how we discuss.

**Comment by wu-sheng** [comment, marked as answer]
Duplicate #7484
